master/internal/db/postgres_experiments_intg_test.go

```diff
@@ -976,3 +976,123 @@ func validateExperimentMatch(t *testing.T, expected []*model.Experiment, actual
 	// map should be empty
 	require.Equal(t, len(m), 0)
 }
+
+func TestExperimentTotalStepTime(t *testing.T) {
+	ctx := context.Background()
+
+	require.NoError(t, etc.SetRootPath(RootFromDB))
+	db := MustResolveTestPostgres(t)
+	MustMigrateTestPostgres(t, db, MigrationsFromDB)
+
+	t.Run("invalid experiment, return 0.0, no error", func(t *testing.T) {
```

Comment:
We cannot create a test coverage that tests the `err` generation of `ExperimentTotalStepTime` function as `SELECT COALESCE..` by default returns `0.0` in such case

Reply:
This coverage looks great. I don't think we need to add any more tests.